src/Tizen.NUI/src/internal/WebView/WebView.cs

```diff
@@ -254,6 +263,16 @@ public WebView(string locale, string timezoneId) : this(Interop.WebView.New2(loc
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Creates an uninitialized WebView.
+        /// <param name="args">args array</param>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public WebView(string[] args) : this(Interop.WebView.New3(args.Length, args), true)
```

Comment:
Can you check the build warning at line 271?
It seems that you need to check nullity of args before referencing it.

Reply:
Done.